code/modules/admin/sql_notes.dm

```diff
@@ -133,15 +160,18 @@
 			output += " <a href='?_src_=holder;shownoteckeylinkless=[target_ckey]'>\[Refresh Page\]</a></center>"
 		output += ruler
 		while(query_get_notes.NextRow())
-			var/id = query_get_notes.item[1]
+			var/secret = text2num(query_get_notes.item[1])
+			if(linkless && secret)
+				continue
```

Comment:
Hidden notes would imbue curiosity, these are intended for what should be actually secret.